src/nupic/algorithms/Cells4.hpp

```diff
@@ -266,9 +266,9 @@ namespace nupic {
         typedef Segment::InSynapses InSynapses;
         typedef std::vector<OutSynapse> OutSynapses;
         typedef std::vector<SegmentUpdate> SegmentUpdates;
-        static const UInt VERSION = 3;
 
       private:
+        static const UInt VERSION = 3;
```

Comment:
no need to be static, and can remain public const?

Reply:
> static const UInt VERSION = 3;
> no need to be static, and can remain public const?

Not doing DLL now so I will back out this change.